docker-compose.yml

```
@@ -32,9 +32,9 @@ services:
       - FLASK_SERVER_PORT=${RPCPORT:-5000}
       - PYTHONUNBUFFERED=1
       - RPCDEBUGPORT=${RPCDEBUGPORT:-5001}
-      - WEBREQUESTPORT=${WEBREQUESTPORT:-5002}
-      - WEBREQUESTHOST=${WEBREQUESTHOST:-localhost}
-      - WEBREQUESTPROTOCOL=${WEBREQUESTPROTOCOL:-http}
+      - RPCDEBUGPORT=${RPCDEBUGPORT:-5001}
+      - WEBDRIVERHOST=${WEBDRIVERHOST:-webdriver}
+      - WEBDRIVERPORT=${WEBDRIVERPORT:-4444}  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)